commit 776fea8f67993c21c160f892328ce086f1bd5201
Author: Hubert Figuière <hub@figuiere.net>
Date:   Sat Aug 13 20:49:58 2022 -0400

    Issue #52 - Fix includedir in pc file

diff --git a/meson.build b/meson.build
index 0a0175c..24e7067 100644
--- a/meson.build
+++ b/meson.build
@@ -64,7 +64,6 @@ pkg.generate(
   description: 'Plugin standard for audio systems',
   filebase: 'lv2',
   name: 'LV2',
-  subdirs: ['lv2'],
   variables: [
     'lv2dir=' + lv2dir,
     'plugindir=' + lv2dir,
